rpm/native_db.go

```diff
@@ -299,3 +426,106 @@ func constructHint(b *strings.Builder, info *Info) string {
 	}
 	return b.String()
 }
+
+func findDBs(ctx context.Context, out *[]foundDB, sys fs.FS) fs.WalkDirFunc {
+	return func(p string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if d.IsDir() {
+			return nil
+		}
+
+		dir, n := path.Split(p)
+		dir = path.Clean(dir)
```

Comment:
I know it's copied over, but why is it `path` instead of `filepath`? Guessing it's inside the TAR?

Reply:
Yeah I suppose so too, maybe related to https://github.com/quay/claircore/issues/1327

Reply:
It's path because layer paths should always use normal slashes, at least to my reading of the [spec](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/pax.html#tag_20_92_13_06). In practice, Windows layers also seem to use normal slashes.